public/views/content/studentView.html

```diff
@@ -0,0 +1,3 @@
+<a ui-sref=".addEntry">+ Add new entry</a>
```

Comment:
Okay. The text links now toggle the forms in and out. It took some doing with ng-switches and it's a little gross, but still better than ng-show and ng-hide I think.